chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -1309,7 +1305,8 @@ impl PeerManagerActor {
             }
             Err(find_route_error) => {
                 // TODO(MarX, #1369): Message is dropped here. Define policy for this case.
-                self.network_metrics.inc(
+
+                NETWORK_METRICS.inc(
```

Comment:
If we do it like this - do we still have a way to checking these values in unittests ? (especially as multiple tests can run in parallel)

Reply:
> If we do it like this - do we still have a way to checking these values in unittests ? (especially as multiple tests can run in parallel)

That has been an issue with Prometheus metrics for a while, including other packages. 
We have a few solutions:
- run some tests with `#[serial]`
- rewrite how we use promethius for tests

Note that, those values were simply broken for unit tests, only the first instance would get initialized properly. With this change "nothing gets more broken than it already is".